services/web/server/src/simcore_service_webserver/folders/_folders_db.py

```diff
@@ -467,6 +463,60 @@ async def get_projects_recursively_only_if_user_is_owner(
         return [ProjectID(row[0]) async for row in result]
 
 
+async def get_all_folders_and_projects_recursively(
+    app: web.Application,
+    connection: AsyncConnection | None = None,
+    *,
+    folder_id: FolderID,
+    private_workspace_user_id_or_none: UserID | None,
+    product_name: ProductName,
+) -> tuple[list[FolderID], list[ProjectID]]:
```

Comment:
since you are not creating a Namedtuple, at least put what you get in the functio name

`get_all_folders_and_projects_recursively` -> `get_all_folders_and_projects_ids_recursively`